### PR TITLE
MIR-1359 fix choose publication button in related item

### PR DESCRIPTION
**Pull request by Antoniafriedrich**
[Link to jira](https://mycore.atlassian.net/browse/MIR-1359).

### CODE REVIEW

**Review by eckstueck — APPROVED** [review]

File: mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js
Comment (golsch) [inline mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js:1]: Why was the indentation changed? 2-space indentation is actually the standard for JavaScript.

**Review by golsch — CHANGES_REQUESTED** [review]

File: mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js
Comment (Antoniafriedrich) [reply mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js:1]: You are correct. It seems I hadn't the codestyle settings for mycore javascript imported correctly for some reason. I imported them and autoformated again.

Comment (golsch) [reply mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js:1]: It is currently still at 4 spaces. According to the documentation, that seems to be okay, but we should possibly question it again. Could you perhaps split the commit and first commit the fix? That way, the changes will be much easier to follow for the review.

Comment (Antoniafriedrich) [reply mir-module/src/main/resources/META-INF/resources/js/mir/relatedItem-modal.js:1]: Sure, the diffs are shown a bit confusingly. The only change is in line 5-7 with a new if statement, which is closed in line 24. The rest is just formatting according to mycore-javascriptstyle.xml.

**Review by golsch — COMMENTED** [review]
The fix is fine, but the new formatting or guidelines for formatting should be reconsidered.

**Pull request merged**